### PR TITLE
feat(ui): upgrade react-components to 0.34.0

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Upgrade to react-components to 0.34.0.
- Remove vanilla override for chips.

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the machine list, select some machines that have tags and open the tag form.
- Check that the chips appear evenly spaced in the rows.

### CODE REVIEW

File: ui/src/app/machines/components/MachineHeaderForms/ActionFormWrapper/TagForm/TagChip/TagChip.test.tsx [hunk 1]
```diff
@@ -16,10 +16,7 @@ const tagMap = new Map([
...
 it("displays chips with counts", () => {
   render(<TagChip machineCount={10} tag={tags[0]} tagIdsAndCounts={tagMap} />);
-  // These tests can't select by role and name because the text is inside an
-  // inner span which React Testing Library doesn't support and we can't
-  // currently apply additional aria labels to the button until react-components
-  // is updated with:
-  // https://github.com/canonical-web-and-design/react-components/pull/727
-  expect(screen.getByText("chip1 (2/10)")).toBeInTheDocument();
+  expect(
+    screen.getByRole("button", { name: "chip1 (2/10)" })
```
Comment (petermakowski) [inline ui/src/app/machines/components/MachineHeaderForms/ActionFormWrapper/TagForm/TagChip/TagChip.test.tsx:20]: 🥇

**Review by petermakowski — APPROVED** [review]
LGTM

**Comment by petermakowski** [comment]
@huwshimi  Not sure this is supposed to look like this (I'm comparing this to the spec https://zpl.io/w2pgNb3)?

### This PR
image
`MAAS/r/machine/sdqtg4/summary`

**Comment by huwshimi** [comment]
> @huwshimi Not sure this is supposed to look like this (I'm comparing this to the spec https://zpl.io/w2pgNb3)?
> 
Oh hmm.. did you install the latest deps (sorry I neglected to add that to my QA instructions)? That's what I would have expected it to look like without the changes from the latest react-components release.

**Review by Caleb-Ellis — APPROVED** [review]
Looks good on my end

**Comment by huwshimi** [comment]
> Looks good on my end

Thanks @Caleb-Ellis. @petermakowski I'm going to land this to unblock me, but if you can reproduce with the latest react-components installed then let me know and I'll investigate further.

**Comment by petermakowski** [comment]
> > @huwshimi Not sure this is supposed to look like this (I'm comparing this to the spec https://zpl.io/w2pgNb3)?
> 
> Oh hmm.. did you install the latest deps (sorry I neglected to add that to my QA instructions)? That's what I would have expected it to look like without the changes from the latest react-components release.

Yeah I did install the dependencies and run `yarn ui`. Must've been a caching issue. It's fine on master now.

**Pull request merged**